modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java

```diff
@@ -129,7 +131,7 @@ public void shouldCheckEnterpriseLicense() {
         String expectedImageName = "neo4j:4.4-enterprise";
 
         assertThatExceptionOfType(IllegalStateException.class)
-            .isThrownBy(() -> new Neo4jContainer<>("neo4j:4.4").withEnterpriseEdition())
+            .isThrownBy(() -> new Neo4jContainer<>("library/neo4j:4.4").withEnterpriseEdition())
```

Comment:
Using `withEnterpriseEdition()` will still use `neo4j:4.4-enterprise` in order to avoid breaking changes when checking if license was accepted.